std/container/array.d

```diff
@@ -1986,7 +1986,7 @@ if (is(Unqual!T == bool))
             _store._backend.length = _store._backend.length - 1;
         }
     }
-    /// ditto
+    ///ditto
```

Comment:
Actually, that one's unrelated to https://github.com/D-Programming-Language/dmd/pull/4745, but I tossed it in because I noticed the "ditto" is showing up in the generated docs ( http://dtest.thecybershadow.net/artifact/website-1d618eaba1a0a418cbe85589a97781591809b3fa-5a06ed4863a1b8ed5de4d7e83de8bae8/web/library-prerelease/std/container/array/array.remove_back.html , near the bottom of the page, in the last "Complexity" section).

But now that I look at the test results for this PR ( http://dtest.thecybershadow.net/results/efe54d2753dcaecdd25f95ec40132bc0d15ff2c1/a53d6f3607852ae15236d10fae4dbacdc86a24fb/ ), removing the space doesn't appear to have made any difference, so I'll revert this change.